tests/test_model.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+# import numpy as np
```

Comment:
Why is this whole file commented out?

Reply:
added tests now, they were not present when PR was opened.